CHANGELOG.md

```diff
@@ -30,7 +31,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 - Do not freeze mkdocs-material to version 4.6.1
 - Remove pre-commit dependency
-
+- Update PyMongo to 3.11 (from 3.7)
```

Comment:
This line should be removed

Reply:
Reworded to: "Remove calls to PyMongo functions marked as deprecated (as of version 3.7)."

Reply:
I saw! the line I removed was a duplicated entry in the changelog, under the last scout version (4.23)